runelite-client/src/main/java/net/runelite/client/plugins/chatnotifications/ChatNotificationsPlugin.java

```diff
@@ -172,6 +172,18 @@ public void onChatMessage(ChatMessage chatMessage)
 					notifier.notify(Text.removeTags(chatMessage.getName()) + ": " + chatMessage.getMessage());
 				}
 				break;
+			case PRIVATECHATOUT:
+				return;
+			case MODCHAT:
+			case PUBLICCHAT:
+			case FRIENDSCHAT:
+			case AUTOTYPER:
+			case MODAUTOTYPER:
+				if (client.getLocalPlayer() != null && Text.toJagexName(Text.removeTags(chatMessage.getName())).equals((client.getLocalPlayer().getName())))
```

Comment:
One too many parens around the localplayer name call
```suggestion
				if (client.getLocalPlayer() != null && Text.toJagexName(Text.removeTags(chatMessage.getName())).equals(client.getLocalPlayer().getName()))
```